### DISCUSSION TITLE
[object Object] 값이 나오는 문제

### DISCUSSION THREAD

**Discussion by Taewook1212** [FRONTENDSCHOOL8/popcorn-euid#183] [Ideas]
보기 편하게 console 이쁘게 정리해서 문자 넣으려다 발생한 문제
 console.log('productData : ' + productData);   이렇게 + 넣으면 typeOf 함수? 처럼 [object] 처럼 나와요
  console.log('productData : ' , productData);  하실 거면 , 쉼표로 구분해주세요..
  
  정말 무시무시한 코드에요
  
  console.log(+); 진짜 무시무시한게 뭐냐면 
여기 안에서 아예 [object] 로 값을 변환시켜버려서 
다음에 코드에 영향을 줘요
  
  삽질...

**Comment by Taewook1212** [comment]
갑자기 이렇게 나오는 ..

**Reply by vHwav** [reply]
앗 이거 해결하신거죠?

**Reply by Taewook1212** [reply]
경화님이 해결해주셨습니다.
갓경화님
